### PR TITLE
OSDOCS-4665: breaking up restricted network update content

### PR DESCRIPTION
**Pull request by skopacz1**
[OSDOCS-4665](https://issues.redhat.com/browse/OSDOCS-4665)

**Versions**: 4.8+ (**NOTE**: This specific PR is only for 4.12 and 4.13)
4.11 PR: #54440 
4.10 PR: #54420 
4.9 PR: #54520
4.8 PR: #54589

This PR takes the existing restricted network update page and breaks the content up into three separate pages/assemblies: an about page, and a page each for the two different methods of performing this process. All OSDOCS links/xrefs to the former page have been updated appropriately. The scope of this PR/work is purely reorganization; very few typo changes have been made to the content itself, and the only "new" info added is a list of prereqs for the OSUS page.

Versions 4.11 and 4.12 both have content for the oc mirror plugin, and the content changes should be identical. Versions 4.8-4.10 do not have content for oc mirror.

QE approved (see comments below)

**Doc Previews**
4.12: https://54298--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster/index.html
4.8-4.11: linked in their respective PRs

### CODE REVIEW

**Comment by skopacz1** [comment]
Hey guys! I'm tagging you in this PR because it will update xrefs in documentation that you own. Let me know if you have any questions, concerns, or whether the changes look good to you. Thanks a bunch!

@bscott-rh : [installing/disconnected_install/installing-mirroring-disconnected.adoc](https://github.com/openshift/openshift-docs/pull/54298/files#diff-43fc804c1fa580ed1b5e7852a4fa7719f7538b21fd6369889e1b7c1b7a695c92)

@bergerhoffer : [security/container_security/security-hosts-vms.adoc](https://github.com/openshift/openshift-docs/pull/54298/files#diff-1dc7543d9321397075094a3757ef0059c968e71ca24933ab4c0437a3c8049d78)

@xenolinux (I know nobody owns the Getting Started section, but @kalexand-rh told me you were the last to work on it): [getting_started/openshift-overview.adoc](https://github.com/openshift/openshift-docs/pull/54298/files#diff-0308caf049c60e2e58b20b3e8ddc2200b01c13a92d4bda39060fbbf1f53fb84b)

@opayne1 (as per our conversation, I'll wait until after 4.12 GA to merge to avoid potential merge conflicts with your own changes): [welcome/index.adoc](https://github.com/openshift/openshift-docs/pull/54298/files#diff-40d72ccc658b0f56a1a01387cffdf2a793c5cd21234a0fcbfcc25cf880d318c5)

**Comment by skopacz1** [comment]
@bergerhoffer whoops, sorry for misattributing the disconnected page ownership! Thanks for taking a look

File: welcome/index.adoc [hunk 1]
```diff
@@ -302,7 +302,7 @@ There is a separate process for
 xref:../updating/updating-disconnected-cluster.adoc#updating-disconnected-cluster[updating a cluster on a restricted network].
 ////
 
-- **xref:../updating/updating-restricted-network-cluster.adoc#update-service-overview_updating-restricted-network-cluster[Understanding the OpenShift Update Service]**: Learn about installing and managing a local OpenShift Update Service for recommending {product-title} updates in restricted network environments.
+- **xref:../updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc#update-service-overview_updating-restricted-network-cluster-OSUS[Understanding the OpenShift Update Service]**: Learn about installing and managing a local OpenShift Update Service for recommending {product-title} updates in restricted network environments.
```
Comment (opayne1) [inline welcome/index.adoc:305]: welcome update lgtm!

File: getting_started/openshift-overview.adoc [hunk 2]
```diff
@@ -104,7 +104,7 @@ be reviewed by cluster administrators and xref:../operators/admin/olm-adding-ope
 
 * **xref:../scalability_and_performance/scaling-cluster-monitoring-operator.adoc#scaling-cluster-monitoring-operator[Scale] and xref:../scalability_and_performance/using-node-tuning-operator.adoc#using-node-tuning-operator[tune] clusters**: Set cluster limits, tune nodes, scale cluster monitoring, and optimize networking, storage, and routes for your environment.
 
-* **xref:../updating/updating-restricted-network-cluster.adoc#update-service-overview_updating-restricted-network-cluster[Understanding the OpenShift Update Service]**: Learn about installing and managing a local OpenShift Update Service for recommending {product-title} updates in restricted network environments.
+* **xref:../updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc#update-service-overview_updating-restricted-network-cluster-OSUS[Understanding the OpenShift Update Service]**: Learn about installing and managing a local OpenShift Update Service for recommending {product-title} updates in restricted network environments.
```
Comment (xenolinux) [inline getting_started/openshift-overview.adoc:107]: Changes LGTM, thank you!

File: updating/updating-restricted-network-cluster/index.adoc [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+If multiple clusters are present within the restricted network, mirror the required release images to a single container image registry and use that registry to update all the clusters.
+
+== Performing a restricted network update
+
+You can use one of the following procedures to update a restricted network {product-title} cluster:
+
+* xref:restricted-network-update-OSUS.adoc#updating-restricted-network-cluster-OSUS[Restricted network updates using the OpenShift Update Service]
+
+* xref:restricted-network-update.adoc#updating-restricted-network-cluster[Restricted network updates using the `oc` command-line interface (CLI)]
```
Comment (LalatenduMohanty) [inline updating/updating-restricted-network-cluster/index.adoc:22]: OC CLI can be used to update the cluster even if there is OSUS running for the disconnected environment.So the categorization is not correct .  Also `Restricted network updates using the OpenShift Update Service` might be misleading because connected clusters update using OpenShift update service. 

The right categorization would be 

1.  Updating disconnected environments when running OSUS in the disconnected environment
2.  Updating disconnected environments without running OSUS in the disconnected environment

**Review by LalatenduMohanty — CHANGES_REQUESTED** [review]

File: updating/updating-restricted-network-cluster/index.adoc (hunk 3 above)
Comment (skopacz1) [reply updating/updating-restricted-network-cluster/index.adoc:22]: Sounds good. I was using the previous headings for these pages, so I'll update them to these titles or something similar.

File: modules/update-service-overview.adoc [hunk 4]
```diff
@@ -4,7 +4,7 @@
 // * updating/updating-cluster-within-minor.adoc
 // * updating/updating-cluster-cli.adoc
 // * updating/updating-cluster-rhel-compute.adoc
-// * updating/updating-disconnected-cluster.adoc
+// * updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc
```
Comment (wking) [inline modules/update-service-overview.adoc:7]: I personally prefer the flexible gray areas "restricted network" hints at, but #48887 is moving the docs-consensus shorthand to "disconnected", so we probably want to stick with that for filenames.

File: updating/updating-restricted-network-cluster/index.adoc [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+:context: about-restricted-network-updates
+
+toc::[]
+
+A restricted network environment is one in which your cluster nodes cannot access the internet. 
+For this reason, you must populate a registry with the installation images. 
+If your registry host cannot access both the internet and the cluster, you can mirror the images to a file system that is disconnected from that environment and then bring that host or removable media across that gap. 
+If the local container registry and the cluster are connected to the mirror registry's host, you can directly push the release images to the local registry.
+
+If multiple clusters are present within the restricted network, mirror the required release images to a single container image registry and use that registry to update all the clusters.
```
Comment (wking) [inline updating/updating-restricted-network-cluster/index.adoc:14]: This is one option, but I don't know if we have a Red Hat preference about how many mirror registries customers chose to use.  So we might want to soften the imperative to:

> A single container image registry may be sufficient to host mirrored images suppling several clusters in the disconnected network.

or some such, to explain that it is possible, but not required, to use a single mirror registry.

File: updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc [hunk 6]
```diff
@@ -0,0 +1,136 @@
...
+
+[id="update-restricted-network-cluster-update-service"]
+
+include::modules/update-service-overview.adoc[leveloffset=+1]
+
+.Additional resources
+
+* xref:../../updating/understanding-upgrade-channels-release.adoc#understanding-upgrade-channels_understanding-upgrade-channels-releases[Understanding upgrade channels and releases]
+
+For clusters with internet accessibility, Red Hat provides over-the-air updates through an {product-title} update service as a hosted service located behind public APIs. However, clusters in a restricted network have no way to access public APIs for update information.
```
Comment (wking) [inline updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc:17]: nit: "provides over-the-air updates" -> "provides over-the-air update recommendations", to make it clear that the update service is just information like "updates to A an B are recommended, while updates to C and D are supported, but possibly not recommended".  The actual updates (changing deployed component version running in the cluster) can be applied with or without that sort of advice.

File: updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc [hunk 7]
```diff
@@ -0,0 +1,136 @@
...
+
+To provide a similar update experience in a restricted network, you can install and configure the OpenShift Update Service locally so that it is available within a disconnected environment.
+
+The following sections describe how to provide over-the-air updates for your disconnected cluster and its underlying operating system.
```
Comment (wking) [inline updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc:21]: nit: I realize "over-the-air" is part of the internal updates-team name, but I'm personally not clear on what it is adding.  Yes, admins can update OpenShift clusters without needing a stack of floppy disks or equivalent.  But that's true of basically all software these days, so possibly drop "over-the-air" throughout these docs?  Or am I missing what "over-the-air" is trying to convey?

File: updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc [hunk 8]
```diff
@@ -0,0 +1,136 @@
...
+
+* Have access to the internet to obtain the necessary container images.
+* Have write access to a container registry in the restricted-network environment to push and pull images. The container registry must be compatible with Docker registry API v2.
+* You must have the `oc` command-line interface (CLI) tool installed.
+* For more information on installing Operators, see xref:../../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace[Installing Operators in your namespace].
+
+[id="registry-configuration-for-update-service"]
+== Configuring access to a secured registry for the OpenShift update service
+
+If the release images are contained in a secure registry, complete the steps in xref:../../registry/configuring-registry-operator.adoc#images-configuration-cas_configuring-registry-operator[Configuring additional trust stores for image registry access] along with following changes for the update service.
```
Comment (wking) [inline updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc:34]: The trust stores are for X.509 certificate authorities, to allow HTTPS and other TLS-based registry connections, etc.  So the cluster deciding whether it should trust the registry.  There are also pull-secrets and similar, for the registry to decide whether it should trust the cluster.  I think "a secure registry" could be tightened down to something like "a registry whose HTTPS X.509 certificate is signed by a custom certificate authority" or some such here (perhaps we have existing language for this in the trust-store docs?), so folks are more likely to click through when they need to, and less likely to click through when they don't need to.  But also, even if we don't ace the send-off wording, I expect readers will eventually figure things out, so no worries if you want to leave the current wording in place.

File: updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc [hunk 9]
```diff
@@ -0,0 +1,136 @@
...
+    ...
+    -----END CERTIFICATE-----
+----
+<1> The OpenShift Update Service Operator requires the config map key name updateservice-registry in the registry CA cert.
+<2>  If the registry has the port, such as `registry-with-port.example.com:5000`, `:` should be replaced with `..`.
+
+include::modules/images-update-global-pull-secret.adoc[leveloffset=+1]
+
+[id="update-service-install"]
+== Installing the OpenShift Update Service Operator
```
Comment (wking) [inline updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc:61]: It is not clear to me why we need an inline section on this, and can't just include "Have the OpenShift Update Service Operator installed" as a pre-requisite.  But if we're looking to limit the immediate change in this pull request, and possibly consider that sort of consolidation in follow-up work, that's fine with me.

File: updating/updating-restricted-network-cluster/restricted-network-update.adoc [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+
+[id="updating-restricted-network-mirror-host"]
+== Preparing your mirror host
+
+Before you perform the mirror procedure, you must prepare the host to retrieve content
+and push it to the remote location.
+
+include::modules/cli-installing-cli.adoc[leveloffset=+2]
+
+// this file doesn't exist, so I'm including the one that should pick up more changes from Clayton's PR - modules/installation-adding-mirror-registry-pull-secret.adoc[leveloffset=+1]
```
Comment (wking) [inline updating/updating-restricted-network-cluster/restricted-network-update.adoc:31]: we may want to figure out what this is about, because "Clayton's PR" seems likely to be a very stale reference that's unlikely to resolve on its own.  But I have no problem punting this investigation to follow-up work, if we don't want to handle it in this pull request.

File: updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc (hunk 6 above)
Comment (skopacz1) [reply updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc:17]: Implementing this feedback, and saving the rest of your comments for a future PR that focuses on content changes. Thank you!

**Comment by skopacz1** [comment]
@shellyyang1989 Could you give this PR and the linked PRs a QE review? Also, let me know if you don't have time to review before the Lunar New Year, and I can find another QE that might be available during that time. Thanks!

File: updating/updating-restricted-network-cluster/index.adoc [hunk 11]
```diff
@@ -0,0 +1,23 @@
+:_content-type: ASSEMBLY
+[id="about-restricted-network-updates"]
+= About disconnected environment updates
+include::_attributes/common-attributes.adoc[]
+:context: about-restricted-network-updates
+
+toc::[]
+
+A restricted network environment is one in which your cluster nodes cannot access the internet.
```
Comment (shellyyang1989) [inline updating/updating-restricted-network-cluster/index.adoc:9]: We are using both `disconnected` and `restricted network`. I'm trying to understand the difference between them. So when we use `disconnected`, we are talking about the cluster. While we use `restricted network`, we are talking about the network. Am I right?

File: updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc [hunk 12]
```diff
@@ -0,0 +1,136 @@
...
+[id="update-service-prereqs"]
+== Prerequisites
+
+* Have access to the internet to obtain the necessary container images.
+* Have write access to a container registry in the restricted-network environment to push and pull images. The container registry must be compatible with Docker registry API v2.
+* You must have the `oc` command-line interface (CLI) tool installed.
+* For more information on installing Operators, see xref:../../operators/user/olm-installing-operators-in-namespace.adoc#olm-installing-operators-in-namespace[Installing Operators in your namespace].
+
+[id="registry-configuration-for-update-service"]
+== Configuring access to a secured registry for the OpenShift update service
```
Comment (shellyyang1989) [inline updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc:32]: update service -> Update Service

File: updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc [hunk 13]
```diff
@@ -0,0 +1,136 @@
...
+====
+
+include::modules/update-service-install-web-console.adoc[leveloffset=+2]
+
+include::modules/update-service-install-cli.adoc[leveloffset=+2]
+
+include::modules/update-service-graph-data.adoc[leveloffset=+1]
+
+[id="update-service-mirror-release_updating-restricted-network-cluster"]
+== Mirroring the {product-title} image repository
```
Comment (shellyyang1989) [inline updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc:77]: It's not clear to me why we append `repository` in the section name. Other than images, do we mirror the image repo?

**Comment by shellyyang1989** [comment]
Moving `The OpenShift Update Service requires a locally accessible registry containing update release payloads.` in section [Mirroring images using the oc adm release mirror command](https://54298--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster/restricted-network-update-osus#update-service-mirror-release-adm-release-mirror_updating-restricted-network-cluster-OSUS) to section [Mirroring the OpenShift Container Platform image repository](Mirroring the OpenShift Container Platform image repository) would be better.

**Comment by shellyyang1989** [comment]
In the doc [Updating disconnected environments using the OpenShift Update Service](https://54298--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster/restricted-network-update-osus), we don't tell users how to find the updates and how to update the cluster.

**Comment by shellyyang1989** [comment]
The structure looks not that good to me. Can we move the instructions on the [Updating disconnected environments without the OpenShift Update Service](https://54298--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster/restricted-network-update.html) page up until [Pausing a MachineHealthCheck resource](https://54298--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster/restricted-network-update.html) to a separate page as prerequisites?

File: updating/updating-restricted-network-cluster/index.adoc (hunk 11 above)
Comment (skopacz1) [reply updating/updating-restricted-network-cluster/index.adoc:9]: So disconnected environment is the preferred term per #48887, and I changed the headings to that use term in an attempt to accommodate SME feedback while saving content updates for a single PR later down the line.

However, I'll just make the changes now so that every instance of `restricted network` is updated to say `disconnected environment` instead. Thanks!

File: updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc (hunk 13 above)
Comment (skopacz1) [reply updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc:77]: I'm not sure, this was how the section was titled when I moved it over. @LalatenduMohanty or @wking, do you know any context about this?

**Comment by skopacz1** [comment]
@shellyyang1989

> In the doc [Updating disconnected environments using the OpenShift Update Service](https://54298--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster/restricted-network-update-osus), we don't tell users how to find the updates and how to update the cluster.

I also noticed this issue when reading through this document the first time. This is something I was planning to address in future updates to these pages as part of [OSDOCS-4664](https://issues.redhat.com/browse/OSDOCS-4664). Since this is already a longstanding issue with the page, would you be okay with me making this change in a later PR?

>The structure looks not that good to me. Can we move the instructions on the [Updating disconnected environments without the OpenShift Update Service](https://54298--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster/restricted-network-update.html) page up until [Pausing a MachineHealthCheck resource](https://54298--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster/restricted-network-update.html) to a separate page as prerequisites?

I'm a little hesitant to make this large of an organizational change without consulting the dev team first, and my intention for this PR was to limit the scope to just splitting the two update methods up into separate pages, while keeping the content for each method the same as much as possible.

**Comment by skopacz1** [comment]
@shellyyang1989 I'm still applying your feedback to the 4.8 and 4.9 PRs, but let me know what you think for the other PRs!

**Comment by shellyyang1989** [comment]
> I also noticed this issue when reading through this document the first time. This is something I was planning to address in future updates to these pages as part of [OSDOCS-4664](https://issues.redhat.com/browse/OSDOCS-4664). Since this is already a longstanding issue with the page, would it be okay if I made this change in a later PR?

It's fine with me.

> I'm a little hesitant to make this large of an organizational change without consulting the development team first, and my intention for this PR was to limit the scope to just splitting the two update methods up into separate pages, while keeping the content for each method the same as much as possible.

If development team agree with it, and we have a jira issue to track it, it's fine with me.

File: updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc (hunk 7 above)
Comment (skrthomas) [reply updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc:21]: remove mention of over-the-air per Trevor's comment here

**Review by skrthomas — COMMENTED** [review]
The reorg itself is sound structurally. A few of Trevor's comments in the files appear to be unaddressed, so perhaps you can go through and make sure that you address his comments and that will be good.

File: modules/update-service-overview.adoc (hunk 4 above)
Comment (skopacz1) [reply modules/update-service-overview.adoc:7]: I went ahead and updated the actual content to use disconnected instead of restrcited. I'll apply these changes to non user-facing things like filenames, anchors, and other metadata in [OSDOCS-4890](https://issues.redhat.com/browse/OSDOCS-4890). This work entails making over 50 changes to the content, and if I do this before these current PRs are merged, these changes would have to be duplicated across 5 PRs. 

Unless anyone has any objection, I'll save this effort for when I tackle the linked ticket in a single PR.

File: updating/updating-restricted-network-cluster/index.adoc (hunk 5 above)
Comment (skopacz1) [reply updating/updating-restricted-network-cluster/index.adoc]: I'll make this change now

File: updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc (hunk 7 above)
Comment (skopacz1) [reply updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc:21]: I'll remove these mentions now

File: updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc (hunk 8 above)
Comment (skopacz1) [reply updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc:34]: I'll see if I can track down specific wording in the documentation, otherwise I'll try something like "a registry whose HTTPS X.509 certificate is signed by a custom certificate authority" and see if it's not too wordy

File: updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc (hunk 9 above)
Comment (skopacz1) [reply updating/updating-restricted-network-cluster/restricted-network-update-OSUS.adoc:61]: Good idea, though similar to @shellyyang1989 's suggestion to flesh out a new prereq page and simplify the actual updating content, I think I'll defer this change to that effort

File: updating/updating-restricted-network-cluster/restricted-network-update.adoc (hunk 10 above)
Comment (skopacz1) [reply updating/updating-restricted-network-cluster/restricted-network-update.adoc:31]: Added investigation of this comment as a task for [OSDOCS-4668](https://issues.redhat.com/browse/OSDOCS-4668)

**Comment by evakhoni** [comment]
> shellyyang1989
The structure looks not that good to me. Can we move the instructions on the [Updating disconnected environments without the OpenShift Update Service](https://54298--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster/restricted-network-update.html) page up until [Pausing a MachineHealthCheck resource](https://54298--docspreview.netlify.app/openshift-enterprise/latest/updating/updating-restricted-network-cluster/restricted-network-update.html) to a separate page as prerequisites?

have to agree on that one. giving the full instructions over and over again in different pages makes it all way too long to be readable. it even includes a step by step on how to install the openshift cli..  
imho, it'll be much better structured and on topic if it'll just reference the relevant pages instead of including them all inline

> skopacz1
I'm a little hesitant to make this large of an organizational change without consulting the development team first, and my intention for this PR was to limit the scope to just splitting the two update methods up into separate pages, while keeping the content for each method the same as much as possible.

also a valid point. while I do feel the suggestion above is important for improving our doc readability, it will indeed require much more changes than originally planed, therefore may or may not be the scope of this PR

**Comment by skopacz1** [comment]
@evakhoni I'm happy to accept suggestions and include them as part of my [OSDOCS-4664](https://issues.redhat.com/browse/OSDOCS-4664) story! Making a common prereq page for disconnected updates will become a subtask of this story as well, in order to further improve readability and organization like you said.

But yeah, this PR is just meant to cover one subtask of that story, breaking up the current page into 3 smaller pages. For that reason, and because any suggestions would have to be implemented 5 times now rather than once in a future PR, I'm doing my best to keep the scope of this PR to just breaking existing content up as well as other major issues that block the reorganization.

**Comment by skopacz1** [comment]
@stevsmit 

> I think that the content looks good, but is there any reason we're removing all instances of Restricted, but not changing module titles from restricted -> disconnected?

The suggestion to update the wording came in after I had already broken up the work into 5 PRs. So I was okay with accommodating that feedback towards user facing content in these PRs, since there were only a few mentions of "restricted" in paragraphs or page titles. 

However, it seems like there are a **lot** of mentions of "restricted" in things like the filenames and anchors. Since these remaining mentions don't affect how users read the actual content, I'd prefer to change all of them in one PR via [OSDOCS-4890](https://issues.redhat.com/browse/OSDOCS-4890) rather than duplicating the work 5 times in these PRs. Would that sound good to you?

**Comment by bergerhoffer** [comment]
> I think that the content looks good, but is there any reason we're removing all instances of Restricted, but not changing module titles from restricted -> disconnected?

@stevsmit @skopacz1 I don't think we should change the file names/anchors. Usually when we do a rename of things for things that are currently published, we don't touch file names, anchors, etc. because that could break bookmarks/other existing links that already point to those sections. I think that updating the actual text should be sufficient in this case.
